### PR TITLE
fix bug #810 and more consistent variable names

### PR DESCRIPTION
**Pull request by nacimgoura**
I fixed a blocking bug when using the api steam. I also renamed a variable to make it more consistent for developers.

### CODE REVIEW

File: views/api/steam.pug [hunk 1]
```diff
@@ -30,11 +30,12 @@ block content
...
-  ul.lead.list-unstyled
-    for achievement in playerAchievemments.achievements
-      if achievement.achieved
-        li.text-success= achievement.name
+  if playerStats
+    h3 #{playerStats.gameName} Achievements
+    ul.lead.list-unstyled
+      for achievement in playerStats.achievements
+        if achievement.achieved
+          li.text-success= achievement.name
```
[the remark this answers is not in the document]
Comment (nacimgoura) [reply views/api/steam.pug]: It's done :wink:

**Comment by YasharF** [comment]
I'm testing this, it looks like it is broken now :(  Trying to see why.

**Comment by YasharF** [comment]
There is a scoping issue.  When testing with two different steam accounts, if I remove the curly brackets that are added on line 290 and 308, the achievements show up and if there aren't any I get "No player achievements".  But if I put them back in, the object `body` doesn't make it out of the chain to be return and I end up with an undefined for playerAchievements at `playerStats: playerAchievements ? playerAchievements.playerstats : null,` and an output of "No player achievements" all the time.

**Comment by nacimgoura** [comment]
I have made a change but I can't test because I don't have any achievements on my steam account. Can you tell me if this is resolved?
If you remove the curly brackets that are added on line 290 and 308, syntax is false.

**Comment by YasharF** [comment]
Considering that we have gone back and forth a few times and that there is still some small work required to merge this, I incorprated the fix for the related bug with another Steam related issue in https://github.com/sahat/hackathon-starter/commit/525e089a053bfcfd4d5cdfce8008609f4724820f

Thanks for bringing this up.

**Comment by nacimgoura** [comment]
so it still didn't work with my changes ?

**Pull request closed without merging**